api/lib/infrastructure/files/candidates-import/candidates-import-transformation-structures.js

```diff
@@ -76,13 +77,41 @@ const _TRANSFORMATION_STRUCT_FOR_PIX_CERTIF_CANDIDATES_IMPORT_V1_5 = [
 ];
 
 // ALL
-const TRANSFORMATION_STRUCTS_FOR_PIX_CERTIF_CANDIDATES_IMPORT_BY_VERSION = {
-  1.5: {
-    version: '1.5',
-    transformStruct: _TRANSFORMATION_STRUCT_FOR_PIX_CERTIF_CANDIDATES_IMPORT_V1_5,
-    headers: _getHeadersFromTransformationStruct(_TRANSFORMATION_STRUCT_FOR_PIX_CERTIF_CANDIDATES_IMPORT_V1_5),
-  },
-};
+function getTransformationStructsForPixCertifCandidatesImportByComplementaryCertifications({
+  complementaryCertifications,
+}) {
+  const transformationStruct = [..._TRANSFORMATION_STRUCT_FOR_PIX_CERTIF_CANDIDATES_IMPORT];
+
+  if (featureToggles.isComplementaryCertificationSubscriptionEnabled) {
+    const containsClea = complementaryCertifications.some(
+      (complementaryCertification) => complementaryCertification.name === CLEA
+    );
+    const containsPixPlusDroit = complementaryCertifications.some(
+      (complementaryCertification) => complementaryCertification.name === PIX_PLUS_DROIT
+    );
+
+    if (containsClea) {
+      transformationStruct.push({
+        header: 'CléA Numérique\n("oui" ou laisser vide)',
+        property: 'cleaNumerique',
```

Comment:
Suggestion : appliquer une convention pour montrer que c'est un booléen, par exemple :
```suggestion
        property: 'hasCleaNumerique',
```

Reply:
Done.